portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,22 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Generates a random color.
+ * @returns {string} A color in HSL (hue, saturation, lightness) format
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function generateRandomColor() {
+  const randomHue = Math.random() * 360; // max value 360
+  const saturation =  (Math.random() * 40) + 40 // max value 100
```

Comment:
So you want a random value of saturation in the range [40, 80] and a random value of lightness in the range [60, 90]? Also, just wanted to point out that these values will be floating point numbers in the above ranges - I suppose hsl can accept floating point values for its parameters?

Reply:
Yes, they can take in floating point values! I tried adding floor to each of the lines but it does not have any different behavior.